docs/basic-docs/atomicdex/atomicdex-api.md

```diff
@@ -725,19 +725,31 @@ Set the value of the `mm2` parameter in the [coins](../../../basic-docs/atomicde
 
 For terminal interface examples, see the examples section below.
 
+#### Using AtomicDEX Software on an Qtum Network
+
+The following information can assist the user/developer in connecting AtomicDEX software to the Qtum network:
+
+- Swap smart contract on the Qtum mainnet: [0x2f754733acd6d753731c00fee32cb484551cc15d](https://qtum.info/contract/2f754733acd6d753731c00fee32cb484551cc15d)
+- Swap smart contract on the Qtum testnet: [0xba8b71f3544b93e2f681f996da519a98ace0107a](https://testnet.qtum.info/contract/ba8b71f3544b93e2f681f996da519a98ace0107a)
+
+To use AtomicDEX software on another QRC20-based network, deploy the Etomic swap contract code from the repository linked below. Use of this code requires either an ETH node setup or access to a public service such as [Infura.](https://infura.io/)
```

Comment:
`QRC20-based network` - should be QTUM-based network/fork, QRC20 are just tokens that run on top of QTUM blockchain/network.

`Use of this code requires either an ETH node setup or access to a public service such as [Infura.](https://infura.io/)` - QTUM does not work with ETH node, it should be `QTUM full node setup`, please also check if it is possible to deploy contracts using Electrum server and mention it here if yes.

Reply:
Sorry, I missed it too, thank you :)